### PR TITLE
Initial release of 20th Event Anniversary Skill

### PR DESCRIPTION
**Pull request by Haydrich**
* **Addressed Issue(s)**: None

* **Server Mode**: RE

* **Description of Pull Request**: Initial release of 20th Event Anniversary Skill

### CODE REVIEW

File: db/re/skill_db.yml [hunk 1]
```diff
@@ -18480,6 +18480,18 @@ Body:
...
+    Name: All_EVENT_20TH_ANNIVERSARY
+    Description: 20th Event Anniversary
+    MaxLevel: 1
+    TargetType: Self
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      AllowOnMado: true
+    Range: 1
+    HitCount: 1
```
Comment (Daegaladh) [inline db/re/skill_db.yml:18493]:
```suggestion
```
Range and HitCount are unnecessary, since it's a Self type skill

**Comment by Daegaladh** [comment]
I assume it doesn't need a duration value as the duration is managed by the client?

**Comment by Haydrich** [comment]
> I assume it doesn't need a duration value as the duration is managed by the client?

Yes, 3 seconds is the Cooldown, as the videos of RO showing this.

**Review by Daegaladh — APPROVED** [review]

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -69516,9 +69516,7 @@ Body:
     Type: Card
     SubType: Enchant
     Script: |
-       /* TODO implement skill
        skill "All_EVENT_20TH_ANNIVERSARY",1;
```
Comment (aleos89) [inline db/re/item_db_etc.yml:69519]:
```suggestion
       skill "ALL_EVENT_20TH_ANNIVERSARY",1;
```

File: db/re/skill_db.yml [hunk 3]
```diff
@@ -18480,6 +18480,16 @@ Body:
     SplashArea: 5
     Duration1: 30000
     Status: Killing_Aura
+  - Id: 784
+    Name: All_EVENT_20TH_ANNIVERSARY
```
Comment (aleos89) [inline db/re/skill_db.yml:18484]:
```suggestion
    Name: ALL_EVENT_20TH_ANNIVERSARY
```

File: db/re/skill_db.yml [hunk 4]
```diff
@@ -18480,6 +18480,16 @@ Body:
...
+    Description: 20th Event Anniversary
```
Comment (aleos89) [inline db/re/skill_db.yml:18485]:
```suggestion
    Description: RO 20th Anniversary Firecracker
```

**Review by aleos89 — APPROVED** [review]

File: db/re/skill_db.yml [hunk 5]
```diff
@@ -18480,6 +18480,16 @@ Body:
...
+  - Id: 784
+    Name: ALL_EVENT_20TH_ANNIVERSARY
+    Description: RO 20th Anniversary Firecracker
+    MaxLevel: 1
+    TargetType: Self
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      AllowOnMado: true
+    Cooldown: 3000
```
Comment (Atemo) [inline db/re/skill_db.yml:18492]:
```suggestion
    Cooldown: 3000
    Requires:
      SpCost: 1
```
according to skillinfolist.lub

**Review by Atemo — APPROVED** [review]

**Pull request merged**